### PR TITLE
Make two columns of options in the options dialog

### LINKED ISSUE #2462: Redesign options.lua window layout
As #2375 extends this window beyond 480px it requires a redesign to fit properly in a 640x480 resolution.

This could be achieved by e.g.
- Making two columns
- Grouping some options into a new sub menu
- Making as scrollbar
- Reactive menu layout (layout changes to fullscreen at 480 height)
- Fullscreen-esque menu layout (e.g. like hotkeys, which is windowed at 640x480 and fullscreen at that resolution)

### PR DESCRIPTION
**Pull request by tobylane**
*Fixes #2462*

**Describe what the proposed change does**
- Put half of the options in a second column
- Make the window wider, adjust the title and back button
- Add button for jukebox

### CODE REVIEW

File: CorsixTH/Lua/dialogs/resizables/options.lua [hunk 1]
```diff
@@ -181,59 +182,63 @@ function UIOptions:UIOptions(ui, mode)
...
   -- "Hotkeys" button
-  local hotkey_y_pos = self:_getOptionYPos() + 10
-  self:addBevelPanel(20, hotkey_y_pos, 280, 40, col_bg):setLabel(_S.options_window.hotkey)
-    :makeButton(0, 0, 280, 40, nil, self.buttonHotkey):setTooltip(_S.tooltip.options_window.hotkey)
+  self:addBevelPanel(320, lower_row_y_pos, BTN_WIDTH, 30, col_bg):setLabel(_S.options_window.hotkey)
+    :makeButton(0, 0, BTN_WIDTH, 30, nil, self.buttonHotkey):setTooltip(_S.tooltip.options_window.hotkey)
+
+  -- "Jukebox" button
+  self:addBevelPanel(465, lower_row_y_pos, BTN_WIDTH, 30, col_bg):setLabel(_S.options_window.jukebox)
+    :makeButton(0, 0, BTN_WIDTH, 30, nil, self.buttonJukebox):setTooltip(_S.tooltip.options_window.jukebox)
```
Comment (lewri) [inline CorsixTH/Lua/dialogs/resizables/options.lua:236]: Jukebox relies on TH Data but luckily enough the demo and full game have the necessary files to load that window. I'd had preferred the jukebox in the main menu but I can accept this.

File: CorsixTH/Lua/dialogs/resizables/options.lua [hunk 2]
```diff
@@ -78,7 +79,7 @@ end
 
 
 function UIOptions:UIOptions(ui, mode)
-  self:UIResizable(ui, 320, 500, col_bg)
+  self:UIResizable(ui, 620, 300, col_bg)
```
Comment (lewri) [inline CorsixTH/Lua/dialogs/resizables/options.lua:82]: The width is very close to the min default of 640. Does it look okay in 640x480?

File: CorsixTH/Lua/dialogs/resizables/options.lua [hunk 3]
```diff
@@ -372,6 +377,12 @@ function UIOptions:buttonHotkey()
   self.ui:addWindow(window)
 end
 
+function UIOptions:buttonJukebox()
+  if self.ui.app.config.audio and self.ui.app:isAudioEnabled() then
+    self.ui:addWindow(UIJukebox(self.ui.app))
```
Comment (lewri) [inline CorsixTH/Lua/dialogs/resizables/options.lua:382]: `UIOptions` is aware of `app` without going through `ui`. But maybe your path is better,

File: CorsixTH/Lua/dialogs/resizables/options.lua (hunk 2 above)
Comment (tobylane) [reply CorsixTH/Lua/dialogs/resizables/options.lua:82]: Screenshot 2024-01-20 at 16 38 53

I think that's ok, it's better than the toggle labels and buttons in one column and the four regular buttons in a second column.

**Pull request merged**